daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/ApiCommandPreprocessorSpec.scala

```diff
@@ -162,9 +162,9 @@ class ApiCommandPreprocessorSpec
         validCreate.copy(argument = ValueRecord("", ImmArray("content" -> ValueInt64(42)))) ->
           a[Error.Preprocessing.TypeMismatch],
         // TEST_EVIDENCE: Integrity: ill-formed exercise API command is rejected
-        validExeTemplate.copy(typeId = TemplateOrInterface.Template("Mod:Undefined")) ->
+        validExeTemplate.copy(typeId = "Mod:Undefined") ->
           a[Error.Preprocessing.Lookup],
-        validExeTemplate.copy(typeId = TemplateOrInterface.Interface("Mod:Undefined")) ->
+        validExeTemplate.copy(typeId = "Mod:Undefined") ->
           a[Error.Preprocessing.Lookup],
```

Comment:
Now this is identical to the case above, so we can remove it

```suggestion
```

Reply:
good catch !